pkg/server/methods/textDocumentSymbols.go

```diff
@@ -0,0 +1,31 @@
+package methods
+
+import (
+	"fmt"
+
+	languageservice "github.com/CircleCI-Public/circleci-yaml-language-server/pkg/services"
+	"github.com/segmentio/encoding/json"
+	"go.lsp.dev/jsonrpc2"
+	"go.lsp.dev/protocol"
+)
+
+func (methods *Methods) DocumentSymbols(reply jsonrpc2.Replier, req jsonrpc2.Request) error {
+	params := protocol.DocumentSymbolParams{}
+	if err := json.Unmarshal(req.Params(), &params); err != nil {
+		return reply(methods.Ctx, nil, fmt.Errorf("%s: %w", jsonrpc2.ErrParse, err))
+	}
+
+	res, err := languageservice.DocumentSymbols(params, methods.Cache, methods.LsContext)
+
+	// fmt.Println("Methods DocSymbols", res)
+
+	if err != nil {
+		return reply(
+			methods.Ctx,
+			nil,
+			err,
+		)
+	}
```

Comment:
do we want to show an error in the client side or maybe show nothing ?

Reply:
I feel like it's spam I would rather not do it